environment.yml

```diff
@@ -16,6 +16,7 @@ dependencies:
     - george
     - gwcs
     - h5py
+    - image_registration
```

Comment:
You shouldn't add image_registration. If your code is up-to-date with main, then image_registration should only be required if actually doing WFC3 analyses, in which case it can be installed using `pip install .[hst]` during installation. And we can't just do `pip install image_registration` or put it in the requirements.txt or environment.yml because the latest PyPI version is bugged and we need to use the newest version on GitHub, which is what the setup.py script does

Reply:
There was an old bug that meant that everyone needed to install image_registration, but the current version of the code shouldn't. Let me know if, after updating your branch against main, you still need to install image_registration - if so, I'll make sure that gets fixed

Reply:
Ah, yeah just following the "create a new environment" documentation, it wasn't working without `image_registration` so hopefully this is something that was fixed now

Reply:
I've removed it from the requirements.txt